src/app.js

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env node
+'use strict';
+let fs = require('fs');
+let path = require('path');
+const chalk = require('chalk');
+const fetch = require('node-fetch');
+
+let uri = process.argv[2]
+let lengthProcess = process.argv.length
+let stats = process.argv[3];
+
+
+const controller = (arrLinks) => {
+    if (lengthProcess === 3) {
+        arrLinks.forEach(link => {
+            console.log(chalk`{yellow ${uri} ${link.slice(0, 50)}}`);
+        })
+    } if (stats === '--stats' || stats === '--s') {
+        statsBasic(arrLinks)
+    } if (stats === '--validate' || stats === '--val') {
+        verifyLinks(arrLinks)
+
+    }
+}
+
+
+const fileRoute = () => {
```

Comment:
Trata de evitar las variables globales, esto puede ser confuso a nivel de Scope 
```suggestion
const fileRoute = (parametro) => {
```